environments/icds-cas/postgresql.yml

```diff
@@ -162,3 +221,104 @@ dbs:
       create: False
       django_migrate: False
       hq_acceptable_standby_delay: 0
+  logical:
+    - name: commcarehq_p10
+      host: pgshard10
+      master_host: pgshard0
```

Comment:
I was hoping to do these from the standbys, but you can't actually create a logical publisher from a standby server.